strawberryfields/engine.py

```diff
@@ -377,7 +375,8 @@ def run(self, program, *, compile_options=None, run_options=None):
 
```

Comment:
Is the `run_options` filtering above necessary? Currently the `Operation` subclasses do not use any kwargs, they're all passed through to the backend API functions. I think it'd be easier to agree that `Operations` and backend API funcs have a shared kwarg namespace.

Reply:
I agree, it would make it cleaner and easier to maintain without filtering.

Furthermore, this way backends could introduce new run options, without having to 'whitelist' them here.

Although, it looks like the above might have been added to avoid failing tests, @co9olguy might be able to fill us in with more details.

Reply:
Yes, without dict comprehension filtering, tests were failing because the `Operations` were complaining about receiving unexpected keyword arguments.

Reply:
I tend to agree that, since everything at this stage is not user-facing, then it is probably fine to include `**kwargs` in the signatures for the Operations class rather than having to whitelist everything